onnxruntime/core/optimizer/graph_transformer_utils.cc

```diff
@@ -211,6 +212,10 @@
       const InlinedHashSet<std::string_view> cpu_cuda_rocm_eps = {onnxruntime::kCpuExecutionProvider,
                                                                   onnxruntime::kCudaExecutionProvider,
                                                                   onnxruntime::kRocmExecutionProvider};
+      const InlinedHashSet<std::string_view> cpu_cuda_rocm_dml_eps = {onnxruntime::kCpuExecutionProvider,
+                                                                  onnxruntime::kCudaExecutionProvider,
```

Comment:
>onnxruntime::kCudaExecutionProvider, [](http://example.com/codeflow?start=66&length=36)

[nit] ORT's .clang-format has pretty bad defaults for effective readability, but let's be consistent.